buildenv

```diff
@@ -1,26 +1,29 @@
-JANSSON_VERSION="2.14"
+export JANSSON_VERSION="2.14"
 export ZOPEN_GIT_URL="https://github.com/akheron/jansson.git"
-export ZOPEN_GIT_DEPS="git make m4 autoconf perl libtool automake bash zoslib tar sed coreutils"
+export ZOPEN_GIT_DEPS="git make cmake m4 autoconf perl libtool automake bash zoslib tar sed coreutils"
 export ZOPEN_TYPE="GIT"
 export ZOPEN_RUNTIME_DEPS=""
 export ZOPEN_COMP="clang"
+export ZOPEN_BOOTSTRAP='./autogen'
+export ZOPEN_EXTRA_CFLAGS="-mzos-target=zosv2r4 -D_ALL_SOURCE -D_OPEN_SYS_FILE_EXT=1 -D_AE_BIMODAL=1"
```

Comment:
Does it compile without these extra cflags?

Reply:
It does compile without the extra cflags, did the changes